src/Jeffijoe.MessageFormat/Jeffijoe.MessageFormat.csproj

```diff
@@ -10,11 +10,11 @@
   	<RepositoryUrl>https://github.com/jeffijoe/messageformat.net</RepositoryUrl>
   	<LangVersion>latest</LangVersion>
   	<Nullable>enable</Nullable>
-  	<TargetFramework>netstandard1.1</TargetFramework>
+  	<TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Was forced to  update to netstandard2.0 to consume the source generator. Feel free to evaluate whether this is worth it or not

Reply:
What are we losing by doing this?

Reply:
The [.NET Standard](https://docs.microsoft.com/en-us/dotnet/standard/net-standard) page documents this, but in layman's terms:

- .NET Framework 4.5 through 4.6 (4.5 and 4.5.1 have reached EoL).
- .NET Core 1.x (which has reached EoL).
- Mono 4.5 through 5.3
- All flavors of Xamarin and UWP would have to be upgraded slightly (probably not an issue for most)

For now keeping a single .NET Standard 2.0 target is the best option for supporting both .NET Framework and .NET Core without adding all of the dependencies that .NET Standard 1.x had.

You may also wish to consider the tradeoffs of supporting earlier versions of .NET Framework and improved API support for .NET Standard 2.1 which you could gain at the price of multi-targeting and keeping conditional sections to gracefully degrade for .NET Framework, both of which add complexity. But IMO, abandoning .NET Standard 1.x at this stage is a good decision either way.

Reply:
So TL;DR is that upgrading to 2.0 is what we want to do either way? 😄

Reply:
Maybe. If multi-targeting .NET Framework 4.5 and .NET Standard 2.1, there isn't a huge gap that .NET Standard 2.0 fills (basically .NET Core 2.x and some older mobile platforms). It depends on how high your tolerance for setting up a more complicated build and for dealing with conditional compilation sections is :wink:.

Reply:
I don't mind removing support from EoL targets.